### ISSUE TITLE
Films/videos

### ISSUE THREAD

**Issue by denisnazarov** [sunday-school/cybernetics-club#2]
Making an issue to keep track of films/videos we've brought up in conversations. Feel free to reply with more!

**[All Watched Over by Machines of Loving Grace](https://en.wikipedia.org/wiki/All_Watched_Over_by_Machines_of_Loving_Grace_%28TV_series%29)** - [Adam Curtis](https://en.wikipedia.org/wiki/Adam_Curtis) (recommended by @frnsys)
[Part 1](https://vimeo.com/groups/96331/videos/80799353) [Part 2](http://www.dailymotion.com/video/x2eagvn_all-watched-over-by-machines-of-loving-grace-2-3-the-use-and-abuse-of-vegetational-concepts-2011_auto) [Part 3](http://www.dailymotion.com/video/x2eku4s_all-watched-over-by-machines-of-loving-grace-3-3-the-monkey-in-the-machine-2011_animals)

**Comment by tarngerine** [comment]
should we watch all 3? watched hte first one last night

**Comment by urcades** [comment]
Yeah, I wasn't too sure on that either — will be watching 'em all for conversation's/brain expansion sake!

**Comment by benauthor** [comment]
episode 2 is the one that touches on cybernetics specifically
